test/rulesets/Random/random.jl

```diff
@@ -44,11 +50,6 @@
         end
 
         # Make sure that we do *not* have these set as non_differentiable. as they are differentiable
-        struct Normal
-            μ
-            σ
-        end
-        Random.rand(d::Normal) = d.μ + d.σ*randn()
         @test frule((Zero(), Normal(0.5,2.0)), rand, Normal(0.1,1.5)) === nothing
```

Comment:
The differentials for types tht don't define `+` and `zero` etc is `Composite`
```suggestion
        @test nothing === frule(
            (Zero(), Composite{NormalDistribution}(μ=0.5,σ=2.0),
            rand,
            NormalDistribution(0.1,1.5),
        )
```